rsmtool/notebooks/summary/intermediate_file_paths.ipynb

```diff
@@ -0,0 +1,63 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Links to Intermediate Files"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Click on the hyperlinks below to see the intermediate experiment files output as part of this summary."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from rsmtool.utils import show_files"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
```

Comment:
Same issue as for `rsmcompare`: current paths in reports do not work and I am almost certain in most cases we'll be generating broken links here as the users will move things around.